ddd/set.rb

```diff
@@ -0,0 +1,15 @@
+class Set
+    attr_reader :id
+    attr_accessor :name, :words
+
+    def initialize(id, name)
+        @id = id
+        @name = name
+        @words = []
+    end
+
+    def add_word(word)
+        @words << word
+    end
+
```

Comment:
use the same amount of indentation in all files (change it here from 4 spaces to 2 spaces)

Reply:
sure